### PR TITLE
[DOCS] update readme to reflect default k8s events handling

### PR DESCRIPTION
**Pull request by afirth**
the readme is contradictory - the default value is never

/fixes 235

### CODE REVIEW

**Review by stsantilena — APPROVED** [review]

**Comment by c-nixon** [comment]
@logdnabot test this please

**Comment by c-nixon** [comment]
@logdnabot test this please

**Comment by c-nixon** [comment]
@logdnabot test this please

File: docs/README.md [hunk 1]
```diff
@@ -285,7 +285,7 @@ Take a look at enabling journald monitoring for [Kubernetes](KUBERNETES.md#colle
 
 A Kubernetes event is exactly what it sounds like: a resource type that is automatically generated when state changes occur in other resources, or when errors or other messages manifest across the system. Monitoring events is useful for debugging your Kubernetes cluster.
 
-By default, the LogDNA agent captures Kubernetes events (and OpenShift events, as well, since OpenShift is built on top of Kubernetes clusters).
+By default, the LogDNA agent does not capture Kubernetes events (and OpenShift events, as well, since OpenShift is built on top of Kubernetes clusters).
 
 To control whether the LogDNA agent collects Kubernetes events, configure the `LOGDNA_LOG_K8S_EVENTS` environment variable using on of these two values:
```
Comment (dkhokhlov) [inline docs/README.md:291]: "using on of these" - typo?

**Review by dkhokhlov — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 1 above)
Comment (c-nixon) [reply docs/README.md:291]: Looks like that's a pre-existing one

**Comment by c-nixon** [comment]
Thanks for the contribution! Sorry for the delay. Jenkins wasn't acknowledging the activation comments

**Pull request merged**